src/fidesops/models/privacy_request.py

```diff
@@ -222,6 +231,40 @@ def cache_identity(self, identity: PrivacyRequestIdentity) -> None:
                     value,
                 )
 
+    def persist_identity(self, db: Session, identity: PrivacyRequestIdentity) -> None:
+        """
+        Stores the identity provided with the privacy request in a secure way, compatible with
+        blind indexing for later searching and audit purposes.
+        """
+        identity_dict: Dict[str, Any] = dict(identity)
+        for key, value in identity_dict.items():
+            if value is not None:
+                hashed_value, salt = ProvidedIdentity.hash_value(value)
+                ProvidedIdentity.create(
+                    db=db,
+                    data={
+                        "privacy_request_id": self.id,
+                        "field_name": key,
+                        # We don't need to manually encrypt this field, it's done at the ORM level
+                        "encrypted_value": {"value": value},
+                        "hashed_value": hashed_value,
+                        "salt": salt,
+                    },
+                )
+
+    def get_persisted_identity(self) -> PrivacyRequestIdentity:
```

Comment:
Do we have a way to get by hashed value yet?

Reply:
This method now [uses a static `SALT`](https://github.com/ethyca/fidesops/blob/743-store-identities/src/fidesops/models/privacy_request.py#L584-L596). This way we can consistently hash values we're searching for to see if they exist in a hashed form in the table.

Reply:
Yay, this looks good to me now.